markdowntoc/markdowntoc.py

```diff
@@ -294,7 +302,7 @@ def main():
     if (params['type'] == 'bear'):
         md_text_toc_pairs, identifiers = create_table_of_contents_bear()
     elif (params['type'] == 'github'):
```

Comment:
You probably want to add `params['type'] == 'github' or params['type'] == 'gitlab'` here?